api/proto/relay/relay.proto

```diff
@@ -42,15 +42,19 @@ message GetChunksRequest {
   // All integers are encoded as unsigned 4 byte big endian values.
   //
   // Perform a keccak256 hash on the following data in the following order:
+  // 1. the length of the operator ID in bytes
```

Comment:
Numbering is off: duplicate `1.`s, and the sublists each have duplicate `ii.`s

Reply:
fixed

Reply:
Nope, looks like there are still two `1.` entries